utils/githooks/pre-commit.d/20-flake.sh

```diff
@@ -20,31 +20,40 @@ fi
 echo "Checking uncommitted code with flake."
 git diff -u | flake8 --diff
 
-if ! BRANCH=$(git rev-parse --abbrev-ref HEAD > /dev/null 2>&1); then
-    echo "Failed to determine branch with git rev-parse"
-    exit 1
+if ! BRANCH=origin/$(git rev-parse --abbrev-ref HEAD 2>/dev/null); then
+        echo "Failed to determine branch with git rev-parse"
+        exit 1
 fi
 
-if [ "$BRANCH" = "master" ]
+if [ "$BRANCH" = "origin/master" ]
```

Comment:
(lint) Couldn't find 'fi' for this 'if'. [SC1046]
(lint) Couldn't parse this if expression. Fix to allow more checks. [SC1073]